velox/dwio/common/Options.h

```diff
@@ -338,6 +345,15 @@ class RowReaderOptions {
     return appendRowNumberColumn_;
```

Comment:
Can you remove this getter and change the setter to set `rowNumberColumnInfo_` instead?  Then change DWRF reader to respect the new options as well

Reply:
done